.github/workflows/ci.yml

```diff
@@ -127,7 +127,7 @@ jobs:
         with:
           go-version-file: "go.mod"
       - name: Run e2e tests
-        uses: ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd@6c72dfc4254a3d157bba5c9c45135cb563b813e7
+        uses: ava-labs/avalanchego/.github/actions/run-monitored-tmpnet-cmd@f99df244cb4e187b408de01f85d8200e7fd5da11
```

Comment:
The action got modified on [this line](https://github.com/ava-labs/avalanchego/compare/6c72dfc4254a3d157bba5c9c45135cb563b813e7...f99df244cb4e187b408de01f85d8200e7fd5da11#diff-4b7f0c97dc5fc50f8faf9f267855acd0c273f128cada14456412734ab1e69752L48)